devenv/README.md

```diff
@@ -1,37 +1,39 @@
-This folder contains useful scripts and configuration for...
+# Dev environment setup
 
-* Configuring dev data sources in Grafana
-* Configuring dev & test scenarios dashboards.
-* Creating docker-compose file with DBs and fake data.
+This folder contains useful scripts and configuration so you can:
 
+* Configure dev data sources in Grafana
+* Configure dev and test scenario dashboards.
+* Create docker-compose file with DBs and fake data.
 
-# Dev dashboards and data sources
+
+## Dev dashboards and data sources
```

Comment:
I think this could be left as `## Dashboards and data sources` since the context is a development environment. Or do you think it should say `## Dashboards and data sources for development` for clarity?

Reply:
If we were to add anything, I would suggest a verb.

I think it makes sense here to leave the adjective because we are talking about dashboards and data sources that are specifically developer tools. A list of some or all of them, or a picture, would be even better. Right now it is awfully vague.